config/components/rbac/role.yaml

```diff
@@ -145,6 +145,7 @@ rules:
   resources:
   - jobsets
   verbs:
+  - create
```

Comment:
this must be related to https://github.com/kubernetes-sigs/kueue/pull/7727/files#diff-d3853fca35347a8a9499f4941707f191a6674df29274b49a9b4ec568eacb0605R60

but then should probably be changed in sync `charts/kueue/templates/rbac/role.yaml`

Reply:
This is why https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/kubernetes-sigs_kueue/7727/pull-kueue-verify-release-0-14/1990843246607077376 failed